packages/suite-native-atoms/src/Select/selectTypes.ts

```diff
@@ -0,0 +1,8 @@
+import { CryptoIconName, cryptoIcons, FlagIconName, flagIcons } from '@trezor/icons';
+
+export const isCryptoIconType = (
```

Comment:
Looks nice, but these functions should be moved to icons package, since it could be useful anywhere not only in Select

Reply:
Yeah, I was thinking the same. Good point

Reply:
[c4bbb5d](https://github.com/trezor/trezor-suite/pull/5506/commits/c4bbb5d664ce069fc6198695d54e89ad7df975ce)